lib/at_cmd/at_cmd.c

```diff
@@ -76,6 +76,24 @@ static int open_socket(void)
 	return 0;
 }
 
+/*
+ * Do any validation of an AT command not performed by the lower layers or
+ * by the modem.
+ */
+static int check_cmd(const char *cmd) {
```

Comment:
Opening bracket shall be put in a separate line (only for function definitions).